I'm not yet able to test locally if we can only use yasm.  Recently started getting some build errors which the GHA on this PR is failing with too.

Not too sure what's up yet.  There was a python upgrade recently so possibly related?  I [noticed when there's a go upgrade](https://github.com/wolfi-dev/os/pull/176/files) we do an epoch update for downstream dependencies, perhaps python is the same?

Build error:
```
  File "/usr/lib/python3.11/importlib/metadata/__init__.py", line 563, in from_name
    return next(cls.discover(name=name))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/meson", line 33, in <module>
    sys.exit(load_entry_point('meson==0.64.1', 'console_scripts', 'meson')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/meson", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
                       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/metadata/__init__.py", line 981, in distribution
    return Distribution.from_name(distribution_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/importlib/metadata/__init__.py", line 565, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: No package metadata was found for meson
```

--- a/gd.yaml
+++ b/gd.yaml
@@ -21,7 +21,7 @@ environment:
       - fontconfig-dev
       - freetype-dev
       - libavif-dev
-      - libjpeg-turbo-dev
+      - libjpeg-dev
       - libpng-dev
       - libwebp-dev
       - libxpm-dev

--- a/libavif.yaml
+++ b/libavif.yaml
@@ -21,7 +21,7 @@ environment:
       - aom-dev
       - cmake
       - dav1d-dev
-      - libjpeg-turbo-dev
+      - libjpeg-dev
       - libpng-dev
       - samurai
       - zlib-dev

